src/pages/App.js

```diff
@@ -63,7 +64,32 @@ export default function App() {
       />
       <button type="submit" onClick={handleSubmit}>Search</button>
       <p>
```

Comment:
This issue is from an earlier PR, but this `p` should be changed to a `ul` or `ol`. Please add a class name to it, too, so we can style it later.

Reply:
Okay

Reply:
The new class name changes look good, and you should keep them, but I meant add a class name to the actual `ul` element. CSS selectors cannot select on children, so we need the `ul` to have a className in order to style it.

Also, since `<div className="container">...</div>` is analogous to `<Container> ... </Container>` right now, it may be best to use that component. This way, if any changes are made to the `Container` element, they'll be applied here too.

Reply:
Done